.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
    flutter_tests:
     docker:
-      - image: cirrusci/flutter
+      - image: cirrusci/flutter:2.10.5
```

Comment:
Is this intended or were you testing something?

Reply:
@AliAbdelfattah No, this was only for testing.